webapp/app/forms/validations/date_validations.py

```diff
@@ -40,6 +40,9 @@ def __call__(self, form, field):
 
         if not day or not month or not year: 
             raise ValidationError(self.message_incomplete)
+        
+        if len(year) < 4:            
+            raise ValidationError(self.message_incomplete)
```

Comment:
You could also just add this to the condition before, right? But I guess this was on purpose?

Reply:
Just wanted to separate it from the other If to help readability. Can also move it to the top of the If if it's not more readable.

Reply:
Yes no, it's fine for me, I was just wondering. 👍